src/debugging.js

```diff
@@ -77,9 +77,11 @@ export function getConfig(debugging) {
 }
 config.getConfig('debugging', ({debugging}) => getConfig(debugging));
 
-export function sessionLoader(storage = window.sessionStorage) {
+export function sessionLoader() {
   let overrides;
   try {
+    if (!window.localStorage || !window.sessionStorage) throw "exception";
```

Comment:
is the problem which putting this in the try/catch solves, that merely accessing `window.sessionStorage` or ` window.localStorage` might throw an exception?  

either way, why is line 83 needed?  wouldn't an exception be thrown on lines 84 or 85 if storage wasn't available?

Reply:
Actually, line 84 may not be required as well... if 85 would be changed to:
```js
overrides = JSON.parse(window.sessionStorage.getItem(OVERRIDE_KEY));
```

Reply:
might need a modified version of 84 as mentioned below.